cli/cmd/install.go

```diff
@@ -482,6 +486,10 @@ func (options *installOptions) recordableFlagSet() *pflag.FlagSet {
 		&options.disableHeartbeat, "disable-heartbeat", options.disableHeartbeat,
 		"Disables the heartbeat cronjob (default false)",
 	)
+	flags.StringVar(
+		&options.heartbeatScheduleOverride, "heartbeat-schedule", options.heartbeatScheduleOverride,
+		"A cron scheduling string for running the heartbeat job. (default: daily at now+10 minutes)",
```

Comment:
Would you mind putting a sample value in like `0 0 * * *`?

Reply:
👍